pkg/registry/flowcontrol/ensurer/strategy.go

```diff
@@ -126,7 +126,12 @@ func NewSuggestedEnsureStrategy() EnsureStrategy {
 // NewMandatoryEnsureStrategy returns an EnsureStrategy for mandatory config objects
 func NewMandatoryEnsureStrategy() EnsureStrategy {
 	return &strategy{
-		alwaysAutoUpdateSpecFn: func(_ wantAndHave) bool {
+		alwaysAutoUpdateSpecFn: func(wh wantAndHave) bool {
+			if want, ok := wh.getWant().(*flowcontrolv1beta3.PriorityLevelConfiguration); ok && want.Name == flowcontrolv1beta3.PriorityLevelConfigurationNameExempt {
```

Comment:
Remember that this server can not rely on the validation done by a different server.  The job here is to stomp on anything that this server does not like, including the `Type` field of the Spec.  This function should say to stomp if that Type field does not hold what this server expects.